recipes/llama-cpp/all/conanfile.py

```diff
@@ -69,6 +71,8 @@ def generate(self):
         deps.generate()
 
         tc = CMakeToolchain(self)
+        if self.options.cuda:
+            tc.variables["LLAMA_CUDA"] = True
```

Comment:
```suggestion
            tc.variables["GGML_CUDA"] = True
```

due to the following, in the logs you have provided:
```
CMake Warning at CMakeLists.txt:95 (message):
  LLAMA_CUDA is deprecated and will be removed in the future.

  Use GGML_CUDA instead

```

Reply:
changed to `GGML_CUDA` instead.